gluefactory/datasets/homographies_point_line.py

```diff
@@ -56,7 +56,7 @@ def plot_predictions(pred, data):
     kp_0 = data["view0"]["cache"]["keypoints"][0].cpu().numpy()
     df_0 = data["view0"]["cache"]["df"][0].cpu().numpy()
     angle_0 = data["view0"]["cache"]["line_level"][0].cpu().numpy()
-    angle_0 = np.arctan2(angle_0[1, :], angle_0[0, :])
+    angle_0 = np.arctan2(angle_0[1], angle_0[0])
```

Comment:
Why did the indices change here? (just asking to understand :D )

Reply:
the indexing for next dimension using ' : ' was basically pointless and did not make a difference. So I removed it to keep it the same as line 66.